packages/light-client/src/index.browser.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./index.js";
+
+// To kep the consistent interface as npm package and browser
+// Use consistent names as named exports
+export * as utils from "./utils/index.js";
+export * as validation from "./validation.js";
+export * as transport from "./transport/index.js";
```

Comment:
can this just be moved to index.ts and then we don't need a special index.browser.ts?

Reply:
We don't need to remove the named exports do we? just add the exports to the root also. (But i'd be fine removing the named exports)

Would this would also resolve the comments here? https://github.com/ChainSafe/lodestar/pull/6685/files#diff-41cdca78f8ff15c341a9e4f70ca767aca6127d72c2ec9f681f8f8ced56aff36cR15-R17

Reply:
I agree with @wemeetagain.  Would it also resolve the weirdness here https://github.com/ChainSafe/lodestar/pull/6673#discussion_r1569525267 ?

Reply:
Done.